resources/waiting-queue/waiting-queue.md

```diff
@@ -0,0 +1,60 @@
+# Waiting queue
+
+Visitors queue service is used for managing the visitors queue for the 8x8 video meetings by keeping visitors websocket connections opened and when a moderator opens the meeting, the visitors are notified and allowed to join the meeting.
+The moderators should be able to see the visitors count.
+
+## Authentication
```

Comment:
Mention how the token is transported? I.e as a header in the STOMP connect frame (if I understand correctly)